src/OrchardCore/OrchardCore.Setup.Abstractions/Events/ISetupEventHandler.cs

```diff
@@ -2,24 +2,29 @@
 
 namespace OrchardCore.Setup.Events;
 
+/// <summary>
+/// Defines methods for handling setup events, including performing setup operations and marking their success or
+/// failure.
+/// </summary>
 public interface ISetupEventHandler
 {
     /// <summary>
-    /// Called during the process of setting up a new tenant.
+    /// Performs asynchronous setup operations using the specified context.
```

Comment:
This is just a notification about setup, it doesn't necessarily 'performs' the setup operation.